spec/models/adventure_spec.rb

```diff
@@ -6,16 +6,27 @@
   it { should have_many(:bucket_list_adventures).dependent(:destroy) }
   it { should have_many(:users) }
 
-  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
-  it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
+  it { should have_valid(:name).when("Visit Sue", "Eat alligator") }
+  it { should have_valid(:address).when(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.